pkg/store/tsdb.go

```diff
@@ -58,6 +58,14 @@ func (s *TSDBStore) Info(ctx context.Context, r *storepb.InfoRequest) (*storepb.
 			Value: l.Value,
 		})
 	}
+
+	// Until we deprecate the single labels in the reply, we just duplicate
+	// them here for migration/compatibility purposes.
+	res.LabelSets = []storepb.LabelSet{
+		{
+			Labels: res.Labels,
```

Comment:
I would aim for invariant that if `res.Labels` are empty then `res.LabelSets` is `[]` - it will simplify interface a bit. WDYT?

Reply:
Yes agreed.

Reply:
Done.